docs/types/mapping-types.rst

```diff
@@ -113,14 +116,14 @@ The example below uses ``_allowances`` to record the amount someone else is allo
 .. index:: !iterable mappings
 .. _iterable-mappings:
 
-Iterable Mappings
+Iterar un Mapping
```

Comment:
I'm skeptical of this line as I believe a more correct way of expressing this title would be "Mapping Iterable" given the usage of "Iterable" as an adjective in the original title.

If you have another opinion on this I am open to hearing it.